sentry-android-core/src/test/java/io/sentry/android/core/ActivityFramesTrackerTest.kt

```diff
@@ -150,6 +318,15 @@ class ActivityFramesTrackerTest {
         sut.stop()
     }
 
+    @Test
+    fun `stop resets frameMetricsAggregator`() {
+        val sut = fixture.getSut()
+
+        sut.stop()
+
+        verify(fixture.aggregator, times(1)).reset()
```

Comment:
`l`: `times(1)` is the default.
```suggestion
        verify(fixture.aggregator).reset()
```